lib/openstudio-standards/prototypes/common/objects/Prototype.Model.rb

```diff
@@ -1239,6 +1250,195 @@ def model_add_guestroom_vacancy_controls(model, building_type)
     end
   end
 
+  # Add guestroom ventilation availability schedules based on the thermostat heating setpoint schedules (to infer setback)
+  # Call to this method needs to be place before purge hour implementation of zone ventilation (if any)
+  #
+  # @code_sections [90.1-2019_6.4.3.3.5.2]
+  # @author Xuechen (Jerry) Lei, PNNL
+  # @param model [OpenStudio::Model::Model] OpenStudio Model
+  # @param building_type [String] Building type
+  def model_add_guestroom_vent_sch(model, building_type)
+    return true unless (template == '90.1-2016') || (template == '90.1-2019')
+
+    # Guestrooms are currently only included in the small and large hotel prototypes
+    return true unless (building_type == 'LargeHotel') || (building_type == 'SmallHotel')
+
+    OpenStudio.logFree(OpenStudio::Info, 'openstudio.model.Model', 'Started Adding Guestroom Ventilation Schedules')
+
+    # Define guestroom occupied maps
+    # List of all spaces that represent vacant rooms
+    guestroom_occupied_map = {
```

Comment:
this section should be in the custom prototype section for each building type

Reply:
This is the prototype section.  While I generally would prefer to see this in the custom adjustments under each building type, it is shared and only applies to the prototype buildings.  It can stay as is.